src/method/technology.tex

```diff
@@ -0,0 +1,100 @@
+\section{Technology}\label{sec:technology}
+
+Throughout the development of the project, a number of different tools and publicly available packages were used to ease
+the development process.
+These different technologies were chosen based on the team's needs and the requirements of the project.
+The following sections will describe the different technologies used in the project.
+
+\subsection{Programming stack}\label{subsec:programming}
+
+As the chess game would be hosted on a website, and will require multiple users to interact with each other, the team
+decided to use TypeScript~\cite{typescript} as the main programming language.
+TypeScript is a superset of JavaScript, with the main difference being that TypeScript is statically typed, while
+JavaScript is dynamically typed.
+Typescript can be very useful in larger projects, as it can help catch errors before the code is run.
+While many in the team did not have any prior experience with TypeScript, in comparison to JavaScript, they quickly
+learned the basics of the language and were able to use it effectively.
+
+The frontend of the project was built with React~\cite{react}, which is a JavaScript / TypeScript library for building
+user interfaces.
+It is developed and maintained by Meta, and is used by many large companies.
+React is focused on mobile applications first and foremost, but the team's focus was on the desktop experience.
+
+% textidote: ignore begin % due to node.js and next.js being identified as an url
+The backend was built using Node.js~\cite{node.js}, which allows the team to build the server using JavaScript.
```

Comment:
A little too inaccurate in my opinion, as we are not building servers. Mayby instead say "which allows the team to build server-side components using TypeScript".